test/integration/guest_user_test.rb

```diff
@@ -26,13 +26,13 @@ def setup
   end
 
   test 'a guest user can browse items by category' do
-    # visit root_path
-    # click_link_or_button('Menu')
-    # select 'hot beverages', from: 'item[category]'
-    # assert page.has_content?(item1.name)
-    # assert page.has_content?(category1.name)
-    # save_and_open_page
-    # refute page.has_content?(item2.name)
-    # refute page.has_content?(category2.name)
+    visit root_path
+    click_link_or_button('Menu')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.